plugins/woocommerce-blocks/tests/e2e/tests/page-content-wrapper/page-content-wrapper.block_theme.side_effects.spec.ts

```diff
@@ -36,17 +36,17 @@ const userText = 'Hello World in the page';
 
 templates.forEach( async ( template ) => {
 	test.describe( 'Page Content Wrapper', async () => {
-		test.beforeAll( async ( { requestUtils } ) => {
-			await requestUtils.deleteAllTemplates( 'wp_template' );
-		} );
 		test( `the content of the ${ template.title } page is correctly rendered in the ${ template.title } template`, async ( {
-			admin,
 			page,
+			admin,
 			editorUtils,
 			frontendUtils,
 		} ) => {
 			await admin.visitAdminPage( 'edit.php?post_type=page' );
+
 			await page.getByLabel( `“${ template.title }” (Edit)` ).click();
+			await page.waitForLoadState();
```

Comment:
I don't think this line is actually necessary since `closeWelcomeGuideModal` waits for the `data` API to be available. Can we remove it? (🏓 @gigitux since you added this line)

Reply:
Let's remove it!

Reply:
Removed in https://github.com/woocommerce/woocommerce/pull/46125/commits/af254f4a2d80e581e5b0df0dafc8e255daf66745